frontend/src/components/pages/ForgotPassword.tsx

```diff
@@ -0,0 +1,184 @@
+    if (!emailPattern.test(userEmail)) {
+      setValidUser(false);
+    } else if (sentEmails.some(item => item.email === userEmail)) {
+      setValidUser(true);
+      setSentEmail(false);
+      setSentEmailToUser(true);
+    } else {
+      // make API call to check if user exists
+      setValidUser(true);
+      setSentEmail(true);
+      // send email logic
+      const newEmail: SentEmail = {
+        email: userEmail,
+        timestamp: new Date().getTime()
+      }
+      sentEmails.push(newEmail);
+      localStorage.setItem("sentEmails", JSON.stringify(sentEmails));
+      setTimeout(() => {
+        const updatedSentEmails: SentEmail[] = JSON.parse(localStorage.getItem("sentEmails") || "[]");
+        const filteredEmails = updatedSentEmails.filter(
+          (item: SentEmail) => item.email !== userEmail
+        );
+        localStorage.setItem("sentEmails", JSON.stringify(filteredEmails));
+        setSentEmailToUser(false)
+      }, 60000);
+    }
+  };
+
+  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setValidUser(true)
+    setSentEmail(false);
+    setSentEmailToUser(false);
+    setUserEmaild(e.target.value);
+  };
+
+  return (
+    <Box
+      height="100vh"
+      backgroundImage={[
+        `url(${backgroundMobile})`,
+        `url(${backgroundMobile})`,
+        `url(${background})`,
+        `url(${background})`,
+      ]}
+      backgroundPosition="center"
+      backgroundRepeat="no-repeat"
+      backgroundSize="cover"
+      backgroundColor="blue.700"
+      display="flex"
+      justifyContent="center"
+      alignItems="center"
+      overflowX="hidden"
+    >
+      <Box
+        maxWidth="100vw"
+        width={["90vw", "90vw", "90vw", "40vw", "30vw"]}
+        height={["20vw", "40px", "20vw", "28vw"]}
+        padding={["36px", "36px", "36px", "60px 64px"]}
+        borderRadius="6px"
+        backgroundColor="var(--gray-50, #F7FAFC)"
+        boxShadow="lg"
+        display="flex"
+        flexDirection="column"
+        minHeight="fit-content"
+      >
+        <Heading
+          color="#4A5568"
+          textAlign="center"
+          fontFamily="Poppins"
+          fontSize="28px"
+          fontStyle="normal"
+          fontWeight="900"
+          lineHeight="120%"
+        >
+          Forgot Password?
+        </Heading>
+        <Text
+          py="36px"
+          color="#4A5568"
+          textAlign="center"
+          fontFamily="Roboto"
+          fontSize="18px"
+          fontWeight="400"
+          lineHeight="150%"
+        >
+          Please enter the email address associated with your account to reset
+          your password.
+        </Text>
+        <Box>
+          <Text
+            color="#4A5568"
+            fontFamily="Roboto"
+            fontSize="18px"
+            fontWeight="400"
+            lineHeight="150%"
+          >
+            Email:
+          </Text>
```

Comment:
maybe you can create a component for this? since it's being used twice and can probably be reused later in the application